src/scripts/verify-revision-is-deployed.sh

```diff
@@ -0,0 +1,55 @@
+# These variables are evaluated so the config file may contain and pass in environment variables to the parameters.
+ECS_PARAM_FAMILY=$(eval echo "$ECS_PARAM_FAMILY")
+ECS_PARAM_SERVICE_NAME=$(eval echo "$ECS_PARAM_SERVICE_NAME")
+ECS_PARAM_CLUSTER_NAME=$(eval echo "$ECS_PARAM_CLUSTER_NAME")
+ECS_PARAM_TASK_DEF_ARN=$(eval echo "$ECS_PARAM_TASK_DEF_ARN")
+
+if [ "$ECS_PARAM_TASK_DEF_ARN" = "" ]; then
+    echo "Invalid task-definition-arn parameter value: $ECS_PARAM_TASK_DEF_ARN"
+    exit 1
+fi
+
+
+if [ -z "${ECS_PARAM_SERVICE_NAME}" ]; then
+    ECS_PARAM_SERVICE_NAME="$ECS_PARAM_FAMILY"
+fi
+
+
+echo "Verifying that $ECS_PARAM_TASK_DEF_ARN is deployed.."
+
+attempt=0
+
+while [ "$attempt" -lt "$ECS_PARAM_MAX_POLL_ATTEMPTS" ]
+
+do
```

Comment:
Formatting seems a little weird to me here. Is it cleaner to inline or multiline?

Reply:
The lines just get a bit long otherwise imo